config/module.config.php

```diff
@@ -2,11 +2,11 @@
 return [
     'service_manager' => [
         'invokables' => [
-            'Dash\Router\Http\Parser\ParserManager' => 'Dash\Router\Http\Parser\ParserManager',
-            'Dash\Router\Http\Route\RouteManager' => 'Dash\Router\Http\Route\RouteManager',
+            Dash\Router\Parser\ParserManager::class => Dash\Router\Parser\ParserManager::class,
```

Comment:
Since @DASPRiD asked me, my opinion is "I don't care" as long as there is a class reference, and not a string reference.

In configs, the extended name is more explicit IMO though.